lib/Pod/Simple/Wiki/Confluence.pm

```diff
@@ -174,7 +174,10 @@ sub _end_L {
 
     # Handle links that are parsed as Pod links.
     if ( defined $link_section ) {
-        $link_target = $link_section;
+        # Other than for L<Link Text>, having a link section defined means
+        # that this is a page anchor link.
+        $link_target = ( $link_attrs->{raw} eq "$link_section" ) ? '' : '#';
```

Comment:
You would probably need to specify a minimum version of Pod::Simple to ensure that `raw` is supported.

However, that generally leads to a lot of complaints from people with Perl installations where they can't upgrade core modules.

See if there is some other way that will work with the versions of Pod::Simple that shipped with older perl versions.